setup.py

```diff
@@ -1,11 +1,28 @@
 from __future__ import absolute_import, division, print_function
 
+import sys
+
 from setuptools import find_packages, setup
 
+# Find pyro version.
+for line in open('pyro/__init__.py'):
+    if line.startswith('__version__ = '):
```

Comment:
Would `pyro.__version__` not work here?

Reply:
This is avoided since the pyro module would requires setup.py would require pyro ...
Also text parsing is faster than loading *all of pyro* as is done in `pyro/__init__.py`.

Reply:
Makes sense.